grizzly/replay/replay.py

```diff
@@ -407,10 +407,13 @@ def run(
                         and self._signature is None
                     ):
                         assert not expect_hang
-                        LOG.debug("no signature given, using %r", short_sig)
+                        LOG.debug("no signature given, using short sig %r", short_sig)
                         self._signature = report.crash_signature
-                        assert not sig_hash, "sig_hash should only be set once"
-                        sig_hash = Report.calc_hash(self._signature)
+                        if self._signature is None:
+                            LOG.debug("failed to generate signature to use")
```

Comment:
I think this should be `warning` ... we want to see in logs if this happens, right?

Reply:
I removed it as discussed since it is already obvious at this point what is happening.